doc/source/whatsnew/v0.21.0.txt

```diff
@@ -413,6 +413,7 @@ I/O
 - Bug in :func:`read_stata` where value labels could not be read when using an iterator (:issue:`16923`)
 - Bug in :func:`read_html` where import check fails when run in multiple threads (:issue:`16928`)
 - Bug in :func:`read_csv` where automatic delimiter detection caused a ``TypeError`` to be thrown when a bad line was encountered rather than the correct error message (:issue:`13374`)
+- Bug in ``DataFrame.to_html()`` with ``notebook=True`` where DataFrames with named indices or non-MultiIndex indices had undesired horizontal or vertical alignment for column or row labels, respectively (:issue:`16792`) 
```

Comment:
I would move this to api_breaking section to give it more visibility. does it warrant a sub-section? @TomAugspurger @jorisvandenbossche

Reply:
I agree it can maybe use a bit more visibility than in here, but I also don't think api breaking section is appropriate, as it is not really an api breakage (I mean, your code won't break because of this), it is just a change (improvement) in the default notebook repr